cmake-modules/acquire_rust.cmake

```diff
@@ -0,0 +1,78 @@
+# CMake function to download the "rustup-init" script and run it to install Rust
```

Comment:
Is this something you authored and thus should carry our copyright?

Reply:
It is, and it should. I'll fix it. But as we discussed, I'm ultimately going to go in a different direction for this.